src/libraries/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.SslCtx.cs

```diff
@@ -49,7 +49,7 @@ internal static byte[] ConvertAlpnProtocolListToByteArray(List<SslApplicationPro
             int protocolSize = 0;
             foreach (SslApplicationProtocol protocol in applicationProtocols)
             {
-                if (protocol.Protocol.Length == 0 || protocol.Protocol.Length > byte.MaxValue)
+                if (protocol.Protocol.IsEmpty || protocol.Protocol.Length > byte.MaxValue)
```

Comment:
This one is also the same size.

https://sharplab.io/#v2:EYLgxg9gTgpgtADwGwBYA0AXEBDAzgWwB8ABAJgEYBYAKGIAYACY8lAbhvqfIDoA5GDNwDKMMAFcoASwwBPdtQ4BmJqQYBhBgG8aDXU2XAZGGAG0AugwCyACgAyk3BgA8Q3ABsAggAcvbyWGwMSQgAOwAFKAgMCEg3AD4GbB8/AKDQiKiYiDdcAEotHT0iyRCMBi9I6NihSQAvGAYAXgY6eSKigDNoGGwwAAsGa1dPZP9A4PDKrLdyqdiGEsTR1ImMquy8wvaC6m3tyQ7BiszY7jXp7gBJXABRfC9ZBkJCWZPss7n32xgQgHMMAYJQzGbiWbAIABq2DcYhguS2e102l2iL2AMiAHcGCEYFiPFBfmJ8D8MDcEGAYA8JtYQthiRAOtYkr4xmlJm8crlcm1UboAL40BGI47rNw1eoMADUzRFF3Op2+fwBUoY5B52wFKO2Qr0xAA7NjcQxgaZZdU6jAzOqGJqigjiAYjKYLLw7A5nMNvCyVulPvElt7xr6OXkdoiSmUzdlxQ1mq0dbourBegMhu4vSkg+zRa8c4tmZm2fKNvCte1kbzdAcjn6PhzuIr/gNGnGni8o2466KGz8mwwgU7QeCoTC4Qny+PEeiIFicXiCUSSWSKVTQjS6TAGUzllni5zuZO9JrD7oOzGpTLa3ue0qBpLVdaira9uP9YasSaTGeLVaEZq+UAA